File: JS/URI/1116.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var caso = parseInt(lines);
j = 1;
for (; caso > 0; caso--) {
    let x = parseFloat(lines[j].split(" ")[0]);
    let y = parseFloat(lines[j].split(" ")[1]);
    let result = x/y;
    console.log((y !== 0) ? ""+result.toFixed(1) : "divisao impossivel");
    j+=1;

}

File: C/URI/1116.c
#include <stdio.h>
 
int main() {
  int x, y, n;
  double result;
        scanf("%d", &n);
        while(n > 0)
        {
                scanf("%d %d", &x, &y);
                if(y == 0)
                        printf("divisao impossivel\n");
                else{
                        result = x / (1.0 * y);
                        printf("%.1lf\n", result);
                }
          n--;

        }


    return 0;
}